internal/characters/thoma/asc.go

```diff
@@ -12,7 +12,7 @@ func (c *char) a1() {
 		if !c.StatusIsActive("thoma-a1") {
 			return 0, false
 		}
-		return float64(c.a1Stack) * 0.05, true
```

Comment:
similar to zhongli, shouldn't need the ifs and should just rely on `a1Stack = 0` at the right times. What concerns me is looking through the implementation we only reset `a1Stack` to 0 on character swap? So if I shield -> get max a1 stack -> stay on thoma and use his skill again it will start at max stacks. Not finding anything on KQM TC for how long these stacks are supposed to last